### ISSUE TITLE
Avoid the flicker when switching the image

### ISSUE THREAD

**Issue by samyuan31515** [openseadragon/openseadragon#1854]
When we switch the display from one image to another, we see a clear flicker. Is there any way to avoid this flicker? It seems to us that some kind of a double-buffer mechanism used in computer graphics community to display smooth animation images may be able to solve the problem. We just wonder whether openseadragon can provide this mechanism. Thanks for your help.

**Comment by iangilman** [comment]
Indeed! The technique you need is to load the next image before removing the previous one. Here's an example:

https://codepen.io/iangilman/pen/erjyBP

… And another with a fade:

https://codepen.io/iangilman/pen/YXRLea

This is a common enough scenario it seems like it might be worth someone creating a plug-in to do it (the core is already big enough, I think features like this need to start migrating to plug-ins), but meanwhile those examples should get you on the right track.

**Comment by puschie286** [comment]
is there any update for v4 ? tried the provided approaches but it seems that preload doesnt work anymore, world contains one entry always

**Comment by iangilman** [comment]
@puschie286 Strange. Preload definitely should still be working! I just tried it now on https://codepen.io/iangilman/pen/erjyBP by updating to v4 in the HTML, like so: 

```
<script src="//cdn.jsdelivr.net/npm/openseadragon@4/build/openseadragon/openseadragon.min.js"></script>
```

... And it seems to be working. Can you share a repro of the issue you're seeing?
